tests/test_acasclient.py

```diff
@@ -555,11 +555,17 @@ def basic_cmpd_reg_load(self, project_code = None, file = None):
                     "required": False,
                     "sdfProperty": "Parent Corp Name"
                 },
+                {
+                    "dbProperty": "Lot Barcode",
```

Comment:
Do we need container inventory enabled to test what this does?

Reply:
Yeah, I flipped container inventory on in acas (not the default) and added this change. I didn't want to change the acas default but I left this change in.